src/gtest/test_tautology.cpp

```diff
@@ -1,10 +1,10 @@
 #include <gtest/gtest.h>
 
-TEST(tautologies, seven_eq_seven) {
+TEST(tautologies, SevenEqSeven) {
     ASSERT_EQ(7, 7);
 }
 
-TEST(tautologies, DISABLED_ObviousFailure)
+TEST(tautologies, DISABLEDObviousFailure)
```

Comment:
Please undo this change. `DISABLED_` is the [documented syntax for disabling a test in GoogleTest](https://github.com/google/googletest/blob/master/googletest/docs/faq.md#how-do-i-temporarily-disable-a-test).